### PR TITLE
Backports 0.18 pr11

### PR DESCRIPTION
**Pull request by Munkybooty**
So something here started breaking PSBT tests, I'm sure there are at least 2 merges responsible for it. Something else broke along the way as well and I have no clue where the error is coming up.

### CODE REVIEW

**Comment by PastaPastaPasta** [comment]
https://github.com/dashpay/dash/pull/4289/commits/1060e0259e80348a351062a796968966548971ba Makes CI not happy. Please resolve. Then mark as ready for review

File: src/rpc/misc.cpp [hunk 1]
```diff
@@ -1143,7 +1143,7 @@ static const CRPCCommand commands[] =
     { "control",            "getmemoryinfo",          &getmemoryinfo,          {"mode"} },
     { "control",            "logging",                &logging,                {"include", "exclude"}},
     { "util",               "validateaddress",        &validateaddress,        {"address"} }, /* uses wallet if enabled */
-    { "util",               "createmultisig",         &createmultisig,         {"nrequired","keys"} },
+    { "util",               "createmultisig",         &createmultisig,         {"nrequired","keys","address_type"} },
```
Comment (UdjinM6) [inline src/rpc/misc.cpp:1146]: 14596: we don't have `address_type`

File: src/wallet/rpcwallet.cpp [hunk 2]
```diff
@@ -4165,7 +4165,7 @@ UniValue getaddressinfo(const JSONRPCRequest& request)
             "  \"ismine\" : true|false,        (boolean) If the address is yours or not\n"
             "  \"iswatchonly\" : true|false,   (boolean) If the address is watchonly\n"
             "  \"isscript\" : true|false,      (boolean) If the key is a script\n"
-            "  \"script\" : \"type\"             (string, optional) The output script type. Only if \"isscript\" is true and the redeemscript is known. Possible types: nonstandard, pubkey, pubkeyhash, scripthash, multisig, nulldata\n"
+            "  \"script\" : \"type\"             (string, optional) The output script type. Only if \"isscript\" is true and the redeemscript is known. Possible types: nonstandard, pubkey, pubkeyhash, scripthash, multisig, nulldata, witness_v0_keyhash, witness_v0_scripthash, witness_unknown\n"
```
Comment (UdjinM6) [inline src/wallet/rpcwallet.cpp:4168]: 14410:
```suggestion
            " \"ischange\" : true|false,       (boolean) If the address was used for change output\n"
            "  \"script\" : \"type\"             (string, optional) The output script type. Only if \"isscript\" is true and the redeemscript is known. Possible types: nonstandard, pubkey, pubkeyhash, scripthash, multisig, nulldata\n"
```

**Review by UdjinM6 — CHANGES_REQUESTED** [review]

File: test/functional/test_runner.py [hunk 3]
```diff
@@ -350,7 +347,7 @@ def main():
         args=passon_args,
         runs_ci=args.ci,
```
Comment (PastaPastaPasta) [inline test/functional/test_runner.py:348]: move this below failfast?

File: .gitignore [hunk 4]
```diff
@@ -145,3 +145,6 @@ make
 # CLion
 .idea
 cmake-build-debug
+
+# Output from running db4 installation
+db4/
```
Comment (PastaPastaPasta) [inline .gitignore:150]: at best we should mark 14515 as partial. IMO we should drop since we don't support openbsd builds

**Review by PastaPastaPasta — CHANGES_REQUESTED** [review]
14596 should be dropped. No point in the empty commit. Just mark as DMN in sheet

**Review by PastaPastaPasta — APPROVED** [review]
utACK for merging via merge commit once CI is happy

**Review by UdjinM6 — APPROVED** [review]
utACK

**Pull request merged**